x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/content/prepackaged_rules/index.ts

```diff
@@ -10,713 +10,713 @@
 // - detection-rules repo using CLI command build-release
 // Do not hand edit. Run script/command to regenerate package information instead
 
-import rule1 from './credential_access_access_to_browser_credentials_procargs.json';
```

Comment:
I noticed that the `scripts/regen_prepackage_rules_index.sh` orders the import by filename, whereas the detection-rules `Package` orders by rule name, which explains the mass change. It should have no impact, but we will see it revert to the name ordering on the next package update.